target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
I would do it in [ltq_mtd_probe()](http://lxr.free-electrons.com/source/drivers/mtd/maps/lantiq-flash.c?v=4.4#L154).

```c
/* useful comment what is done here */
if (ltq_ebu_r32(LTQ_EBU_BUSCON0) & BIT(32)))
	ltq_mtd->map->map_priv_1 = LTQ_NOR_NORMAL;
else
	ltq_mtd->map->map_priv_1 = LTQ_NOR_PROBING;
```

But as long as the meaning of bit 32 of the EBU_BUSCON register is unknow, it is likely that it causes issues for other boards. Hence the request for checking existing code to get what this bit is supposed to do.